modules/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageService.java

```diff
@@ -369,19 +369,18 @@ public void applyClusterState(ClusterChangedEvent event) {
                     .filter(entry -> entry.getValue().getCredential() != null)
                     .collect(Collectors.toUnmodifiableMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
Nit: Could we change `clientSettings` to be `clientSettingsWithCredentials` to be more descriptive?